app/representers/api/base_representer.rb

```diff
@@ -17,4 +17,8 @@ def self.alternate_host
   def self.profile_host
     ENV['META_HOST'] || 'meta.prx.org'
   end
+
+  def index_url_params
```

Comment:
Nice!

Would it also be worth adding a `show_url_params`?  Or maybe we should just wait until there's more than `?zoom` in there.  Stop pre-optimizing, Ryan!